apps/coordinator/src/components/Wallet/TransactionAnalysis/SWASlider.tsx

```diff
@@ -0,0 +1,227 @@
+        config: {
+          scriptType: walletConfig.addressType,
+          requiredSigners: walletConfig.quorum.requiredSigners,
+          totalSigners: walletConfig.quorum.totalSigners,
+        },
+        effectiveFeeRate: feeRateSatPerVb.toNumber(),
+        estimatedLongTermFeeRate: longTermFeeEstimate,
+        hasChange,
+        spendAmount: nonChangeOutputsTotalSats.toNumber(),
+      });
+
+      setWasteAmount(new BigNumber(newWasteAmount));
+    } catch (err) {
+      console.error("Error calculating waste metrics:", err);
+      setWasteAmount(new BigNumber(0));
+    }
+  }, [
+    fee,
+    feeRate,
+    outputs,
+    inputs,
+    changeAddress,
+    walletConfig,
+    longTermFeeEstimate,
+  ]);
+
+  const feeLevelInfo = useMemo(
+    () => getFeeLevelInfo(longTermFeeEstimate),
+    [longTermFeeEstimate],
+  );
+
+  return (
+    <Box className="swa-slider-container">
+      {/* Header */}
+      <Typography variant="h6" className="waste-analysis-title">
+        Waste Analysis
+      </Typography>
+      <Typography variant="body2" className="waste-analysis-description">
+        Analyzes if spending Bitcoin now is efficient or if waiting could be
+        cheaper
+      </Typography>
+
+      {/* Fee Waste Box */}
+      <Box className="potential-fee-waste-container">
+        <Typography variant="subtitle1" className="potential-fee-waste-title">
+          Potential Fee Waste
+        </Typography>
+
+        {/* Slider */}
+        <Box className="slider-container">
+          <Box className="gradient-track" />
+          <Slider
+            value={longTermFeeEstimate}
+            min={1}
+            max={500}
+            step={1}
+            onChange={handleSliderChange}
+            aria-labelledby="long-term-fee-estimate-slider"
+            className="custom-slider"
+            valueLabelDisplay="auto"
+            getAriaValueText={valuetext}
+            valueLabelFormat={valuetext}
+          />
+        </Box>
+
+        {/* Waste Amount */}
+        <Box className="waste-amount-container">
+          <Typography variant="h4" className="waste-amount-text">
+            {formatNumber(wasteAmount)}
+          </Typography>
+          <Typography variant="body2" className="waste-amount-units">
+            sats
+          </Typography>
+          <Tooltip title="Shows how much you might waste in fees by spending now instead of waiting for lower fees">
+            <IconButton size="small" className="info-icon-button">
+              <InfoIcon className="info-icon" />
+            </IconButton>
+          </Tooltip>
+        </Box>
+
+        {/* Fee Estimate Info */}
+        <Typography
+          id="long-term-fee-estimate-slider"
+          variant="body2"
+          className="fee-estimate-text"
+        >
+          Long Term Fee Estimate (L): {longTermFeeEstimate.toLocaleString()}{" "}
+          sats/vB
+          <Tooltip title="Expected fee rate in the future when network activity is lower">
```

Comment:
```suggestion
          <Tooltip title="Expected fee rate in the future. Depending on how long you're able to wait to send this transaction, what do you expect the fees to be. This will help determine if it's better to wait to send this transaction.">
```